packages/contracts/src/plugins/governance/majority-voting/token/TokenVotingSetup.sol

```diff
@@ -201,15 +191,7 @@ contract TokenVotingSetup is PluginSetup {
             revert WrongHelpersArrayLength({length: helperLength});
```

Comment:
This is not needed as the helpers aren't used in the prepareUninstallation function

Reply:
I still prefer to leave it there, the reason being is that if I remove and someone starts to work on the next version of plugin setup, he will not remember that the check is needed which could lead to dangerous activity. So this acts as a remind and doesn't cost much gas.

Reply:
fine